donut/modules/arcfeedback/templates/complaint.html

```diff
@@ -0,0 +1,168 @@
+{% extends "layout.html" %}
+{% block page %}
+<div class="container theme-showcase" role="main">
+    <div class="jumbotron">
+	<div class="row">
+	    {% if 'ADD_REMOVE_EMAIL' in perms %}
+	    {# Leave space for the add/remove email panel #}
```

Comment:
and check if 'ADMIN' in perms

Reply:
I already check for ADMIN in the route -- if a user has ADMIN, the template will be passed all permissions by name